src/components/styles/Theme.tsx

```diff
@@ -0,0 +1,300 @@
+      textSize: typography.sizes.text.body,
+      boxShadowStyle: `0 1px 0 0 ${colors.greyLight}`,
+      borderStyle: `1px solid ${colors.greyLighter}`,
+      backgroundStyle: `${colors.white};`,
+    },
+    secondary: {
+      headingSize: typography.sizes.heading.h6,
+      textSize: typography.sizes.text.small,
+      boxShadowStyle: `0 1px 0 0 ${colors.greyLight}`,
+      borderStyle: `1px solid ${colors.greyLighter}`,
+      backgroundStyle: `${colors.white};`,
+    },
+    brand: {
+      headingSize: typography.sizes.heading.h6,
+      textSize: typography.sizes.text.small,
+      boxShadowStyle: 'none',
+      borderStyle: `1px solid ${colors.brand}`,
+      backgroundStyle: `${colors.brandLight};`,
+    },
+    action: {
+      headingSize: typography.sizes.heading.h6,
+      textSize: typography.sizes.text.small,
+      boxShadowStyle: 'none',
+      borderStyle: `1px solid ${colors.greyLighter}`,
+      backgroundStyle: `${colors.white};`,
+    },
+    info: {
+      headingSize: typography.sizes.heading.h6,
+      textSize: typography.sizes.text.small,
+      boxShadowStyle: 'none',
+      borderStyle: `1px solid ${colors.greyLightest}`,
+      backgroundStyle: `${colors.greyLightest}`,
+    },
+  },
+  errorMessage: {
+    textColor: `${colors.alertDark}`,
+    backgroundColor: `${colors.alertLight}`,
+  },
+  input: {
+    color: colors.grey,
+    borderRadius: `8px`,
+    hover: {
+      border: colors.brand,
+      boxShadow: `0 0 4px 0 ${colors.brand}`,
+    },
+    disabled: {
+      color: colors.greyDark,
+      backgroundColor: colors.greyLightest,
+    },
+    borderColorByStatus: {
+      error: colors.alert,
+      valid: colors.success,
+      disabled: colors.greyLight,
+      default: colors.grey,
+    },
+  },
+  typography: {
+    primary: `"Open Sans", Roboto, Helvetica, Arial, sans-serif`,
+    text: {
+      color: colors.greyDarkest,
+      sizes: {
+        lead: '18px',
+        body: '16px',
+        small: '14px',
+      },
+    },
+    heading: {
+      sizes: {
+        display: '68px',
+        h1: '46px',
+        h2: '38px',
+        h3: '28px',
+        h4: '24px',
+        h5: '18px',
+        h6: '16px',
+      },
+    },
+    lineHeight: {
+      display: '76px',
+      h1: '54px',
+      h2: '46px',
+      h3: '36px',
+      h4: '32px',
+      h5: '26px',
+      h6: '24px',
+      lead: '26px',
+      body: '24px',
+      small: '20px',
+    },
+    letterSpacingMap: {
+      display: '-2.86px',
+      h1: '-1.25px',
+      h2: '-0.85px',
+      h3: '-0.45px',
+      h4: '-0.25px',
+      h5: '-0.02px',
+      h6: '-0.01px',
+    },
+    weights: {
+      regular: 400,
+      semiBold: 600,
+      bold: 700,
+      extraBold: 800,
+    },
+  },
+};
+
+const ThemeContext = React.createContext<AppTheme>(zopaTheme);
+ThemeContext.displayName = 'ThemeContext';
+
+export function useThemeContext() {
+  const context = React.useContext(ThemeContext);
+  if (!context) {
```

Comment:
Is there a way we can avoid this, really keen not to get every single app to update, there will be quite a bit of pushback for this

Reply:
So in theory the error below will never be hit unless somebody somehow uses the ThemeProvider within their app and passes in an empty theme prop. The ThemeContext has a default of the Zopa theme, so the context should always exist.